input_topic: "region_clusters"
out_topic: "/rgb_clusters"